src/sentry/static/sentry/app/views/settings/components/forms/controls/radioGroup.jsx

```diff
@@ -41,7 +40,8 @@ RadioGroup.propTypes = {
   onChange: PropTypes.func,
 };
 
-const RadioLineButton = styled('div')`
+const RadioLineButton = styled('button')`
```

Comment:
I think we'll need to add a background color here to override the default button state: https://percy.io/getsentry/sentry/builds/2334438/view/151476571/1280?browser=firefox&mode=diff

Reply:
fixed